test/unit/calculators/simplified_expenses_checker_calculator_test.rb

```diff
@@ -433,15 +433,75 @@ class SimplifiedExpensesCheckerCalculatorTest < ActiveSupport::TestCase
           @calculator = SimplifiedExpensesCheckerCalculator.new
         end
 
-        should "be true if no_vehicle_emission is yes" do
-          @calculator.no_vehicle_emission = "yes"
+        should "be true if vehicle_emission is low and car_status_before_usage is set to new" do
+          @calculator.type_of_vehicle = "car"
+          @calculator.new_or_used_vehicle = "new"
+          @calculator.vehicle_emission = "low"
+          assert @calculator.vehicle_is_green?
+        end
+
+        should "be true if vehicle_emission is low and new_or_used_vehicle is set to new" do
+          @calculator.type_of_vehicle = "car"
+          @calculator.new_or_used_vehicle = "new"
+          @calculator.vehicle_emission = "low"
           assert @calculator.vehicle_is_green?
         end
 
-        should "be false if no_vehicle_emission is no" do
-          @calculator.no_vehicle_emission = "no"
+        should "be false if type_of_vehicle isn't set to car" do
+          @calculator.type_of_vehicle = "motorbike"
+          @calculator.new_or_used_vehicle = "new"
+          @calculator.vehicle_emission = "low"
           refute @calculator.vehicle_is_green?
         end
+
+        should "be false if vehicle_emission isn't set to low" do
```

Comment:
This is only testing the even where this question is skipped. currently this question only turns update for cars.